tests/unit/test_adapters.py

```
@@ -131,7 +129,9 @@ async def test_url_adapter_rejects_loopback_ip() -> None:
 
     with patch("socket.getaddrinfo") as mock_dns:
         mock_dns.return_value = [(socket.AF_INET, socket.SOCK_STREAM, 0, "", ("127.0.0.1", 80))]
-        with pytest.raises(ValueError, match="[Pp]rivate|[Ll]oopback|[Bb]locked|[Ss]SRF|[Rr]estricted"):
+        with pytest.raises(
+            ValueError, match="[Pp]rivate|[Ll]oopback|[Bb]locked|[Ss]SRF|[Rr]estricted"
```

Ruff (0.15.7)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)